web/test/helpers/helpers-formation.tsx

```diff
@@ -201,7 +221,7 @@ export const createFormationPageHelpers = (): FormationPageHelpers => {
     });
   };
 
-  const getStepStateInStepper = (index: number): string => {
+  const getStepStateInStepper = (index: number | undefined): string => {
```

Comment:
why this change? Why would index be undefined, and if index *is* undefined, why would we be requesting its step state?